fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -706,7 +743,7 @@ private fun BottomDrawer(
                 .clip(drawerShape)
                 .background(drawerBackground)
                 .semantics {
-                    if (drawerState.isOpen) {
+                    if (drawerState.hasOpenedState && !drawerState.isClosed) {
```

Comment:
drawerState.hasOpenedState is not required

Reply:
Done